lambda/facia-responder/src/main.test.ts

```diff
@@ -70,6 +72,102 @@ describe('main', () => {
 			new Date(2024, 0, 2),
 		);
 
+		expect(importCurationDataMock.mock.calls[2][0]).toEqual(
+			JSON.stringify(validMessageContent.fronts['all-recipes']),
+		);
+		expect(importCurationDataMock.mock.calls[2][1]).toEqual('us');
+		expect(importCurationDataMock.mock.calls[2][2]).toEqual('all-recipes');
+		expect(importCurationDataMock.mock.calls[2][3]).toEqual(
+			new Date(2024, 0, 2),
+		);
+
+		expect(importCurationDataMock.mock.calls[3][0]).toEqual(
+			JSON.stringify(validMessageContent.fronts['meat-free']),
+		);
+		expect(importCurationDataMock.mock.calls[3][1]).toEqual('us');
+		expect(importCurationDataMock.mock.calls[3][2]).toEqual('meat-free');
+		expect(importCurationDataMock.mock.calls[3][3]).toEqual(
+			new Date(2024, 0, 2),
+		);
+
+		const notifyFaciaToolMock = notifyFaciaTool as jest.Mock;
+		expect(notifyFaciaToolMock.mock.calls[0][0]).toMatchObject({
+			edition: 'feast-northern-hemisphere',
+			issueDate: '2024-01-02',
+			message:
+				'This issue has been published but its date is in the past so it can only be seen in the Fronts Preview tool',
+			status: 'Published',
+			version: 'v1',
+		});
+	});
+
+	it('should separate US only and rest-of-world fronts', async () => {
+		console.log(JSON.stringify(validMessageContentWithUsOnly));
+
+		const rec = {
+			Records: [
+				{
+					eventSource: 'sqs',
+					awsRegion: 'xx-north-n',
+					messageId: 'BDB66A64-F095-4F4D-9B6A-135173E262A5',
+					body: JSON.stringify(validMessageUsOnly),
+				},
+			],
+		};
+
+		// @ts-ignore
+		await handler(rec, null, null);
+
+		expect(importCurationDataMock.mock.calls.length).toEqual(4);
```

Comment:
Nit: I wonder if representing this state as a 2D array or similar might be a bit more readable?